packages/webapp/webapp_server.js

```
@@ -464,27 +464,21 @@ async function getBoilerplateAsync(request, arch, response) {
-        // Callbacks should return false if they did not make any changes.
-        if (result !== false) {
-          madeChanges = true;
-        }
-      });
-  });
+  for (const key of Object.keys(boilerplateDataCallbacks)) {
+    const callback = boilerplateDataCallbacks[key];
+    const result = await callback(request, data, arch, response);
+    // Callbacks should return false if they did not make any changes.
+    if (result !== false) {
+      madeChanges = true;
```

ESLint

[error] 'madeChanges' is assigned a value but never used.

(no-unused-vars)